webapp/src/features/UserItem/BillingHistory.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Button, Image } from 'react-bootstrap'
+import Cards from '../../widgets/Cards'
+import Table from '../../widgets/Table'
+import Icons from '../../assets/icons'
+import { useDispatch, useSelector } from 'react-redux'
+import { setSelectedUserItemModal } from './userItem.slice'
+import { selectedUserItemModal } from './userItem.selectors'
+import { USERITEM_MODAL_TYPES } from '../UserItemModals/UserItemModals.constants'
+import getUserItemModal from '../UserItemModals'
+
+const BillingHistory = () => {
+	const dispatch = useDispatch()
+	const selectedModal = useSelector(selectedUserItemModal)
+
+	const deleteButtonFormatter = (cell, row) => (
+		<div className='d-inline-flex align-items-center'>
+			{row.revenue === '$115' && (
+				<Button variant='link' className='btn-auto p-0'>
+					<Image
+						src={Icons.alertIcon}
+						alt='alert-icon'
+						width='20'
+						className='ml-2'
+					/>
+				</Button>
+			)}
+			<Button
+				variant='link'
+				className='btn-auto p-0'
+				onClick={() =>
+					dispatch(
+						setSelectedUserItemModal(USERITEM_MODAL_TYPES.DELETE_INVOICE_MODAL)
+					)
+				}
+			>
+				<Image src={Icons.binIcon} alt='bin-icon' width='20' className='ml-2' />
+			</Button>
+			<Button variant='link' className='btn-auto p-0'>
+				<Image
+					src={Icons.arrowRightDarkIcon}
+					alt='arrow-right-dark-icon'
+					width='20'
+					className='ml-2'
+				/>
+			</Button>
+		</div>
+	)
+
+	const mockTableData = [
+		{
+			id: 1,
+			invoice: '#123456789',
+			date: '01/01/22',
+			revenue: '$115',
+		},
+		{
+			id: 2,
+			invoice: '#123456789',
+			date: '01/01/22',
+			revenue: '$105',
+		},
+		{
+			id: 3,
+			invoice: '#123456789',
+			date: '01/01/22',
+			revenue: '$120',
+		},
+		{
+			id: 4,
+			invoice: '#123456789',
+			date: '01/01/22',
+			revenue: '$220',
+		},
+		{
+			id: 5,
+			invoice: '#123456789',
+			date: '01/01/22',
+			revenue: '$50',
```

Comment:
so we are expecting the price thing with '$' sign or we are going to update this thing with formatter in upcoming PRs

Reply:
Actually revenue value will be received as formatted from the backend due to currency being used can be variable.